xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/internal/junit5/servletengine/ServletContainerExecutor.java

```diff
@@ -410,9 +410,8 @@ private GenericContainer<?> createServletContainer() throws Exception
                             // https://downloadarchive.documentfoundation.org/libreoffice/old so that we can benefit
```

Comment:
Seems the comment is not accurate anymore.

Reply:
It is accurate, the full comment is: 
> Note: we use https://download.documentfoundation.org/libreoffice/stable/ and not https://downloadarchive.documentfoundation.org/libreoffice/old so that we can benefit [...]

You might have missed the beginning on the line above.

Reply:
OK, it was not accurate before :)

Reply:
Yep I didn't change the comment when I changed it to use 7.6 with the old url.